spec/lib/phobos/producer_spec.rb

```diff
@@ -129,7 +171,11 @@ class TestProducer1
       end
 
       describe 'with a delivery interval set' do
-        let(:config_hash) { Phobos.config.producer_hash.merge(delivery_interval: 10) }
+        let(:config_hash) do
+          hash = Phobos.config.producer_hash.merge(delivery_threshold: 10)
+          hash.delete(:persistent_connections)
```

Comment:
I dont think you need to do this, now that we `.reject { |k, _| INTERNAL_PRODUCER_PARAMS.include?(k) }`, right?

Reply:
Nope, I still have to do this because it gets passed into the Kafka producer, which crashes on unknown keyword if I leave it in.